client/blocks/signup-form/passwordless.jsx

```diff
@@ -78,22 +78,30 @@ class PasswordlessSignupForm extends Component {
 			form,
 		} );
 
-		const shouldRecordRecaptchaAction =
-			typeof this.props.recaptchaClientId === 'number' &&
-			'onboarding' === this.props.flowName &&
-			'show' === abtest( 'userStepRecaptcha' );
+		const isRecaptchaLoaded = typeof this.props.recaptchaClientId === 'number';
+		const isRecaptchaABTest =
+			'onboarding' === this.props.flowName && 'show' === abtest( 'userStepRecaptcha' );
 
-		const recaptchaToken = shouldRecordRecaptchaAction
-			? await recordGoogleRecaptchaAction(
-					this.props.recaptchaClientId,
-					'calypso/signup/formSubmit'
-			  )
-			: undefined;
+		const recaptchaToken =
+			isRecaptchaABTest && isRecaptchaLoaded
+				? await recordGoogleRecaptchaAction(
+						this.props.recaptchaClientId,
+						'calypso/signup/formSubmit'
+				  )
+				: undefined;
+
+		let recaptchaError = undefined;
+		if ( isRecaptchaABTest && ! isRecaptchaLoaded ) {
+			recaptchaError = 'recaptcha_didnt_load';
+		} else if ( isRecaptchaABTest && isRecaptchaLoaded && ! recaptchaToken ) {
```

Comment:
Style nit: you're checking `isRecaptchaABTest` too often -- why not do it just once and put all the conditional code into one `if` block?